packages/playground-server/src/db.ts

```diff
@@ -125,8 +125,22 @@ export async function matchmakeUser(userToMatch: User): Promise<MatchedUser> {
   });
 }
 
-export async function getUsers(filters: {}): Promise<User[]> {
+export async function getUsers(
+  filters: {},
+  fields: string[] = []
+): Promise<User[]> {
   const db = getDatabase();
+  let returnFields = fields;
```

Comment:
Why not just name the function parameter `returnFields`?

Reply:
Because it's not a good practice to reassign value parameters.